footer.php

```diff
@@ -55,10 +59,10 @@
         <?php if (!empty($mailchimp_url)) { ?>
 
           <div class="col col-l col-l-4">
-            <p>Mailing List</p>
+            <h5 class="margin-bottom-micro">Mailing List</h5>
             <form method="post" id="mailchimp-form" name="mailchimp-form">
-              <input type="email" size="30" value="email" name="email">
-              <input type="submit" value="Subscribe" name="subscribe" class="button">
+              <input type="email" size="30" placeholder="email" value="" name="email">
```

Comment:
And just allowing some random markup to be dropped in via meta field could lead to some fuckery no?